lib/loggly/common.js

```diff
@@ -109,35 +117,75 @@ common.loggly = function () {
   }
 
   var requestOptions = {
-    uri: uri,
+    uri: isBulk ? uri + "/tag/" + headers['X-LOGGLY-TAG'] : uri,
     method: method,
-    headers: headers || {}
+    headers: isBulk ? {} : headers || {}             // Set headers empty for bulk
   };
 
   if (auth) {
     requestOptions.headers.authorization = 'Basic ' + new Buffer(auth.username + ':' + auth.password).toString('base64');
   }
-
-  if (requestBody) {
+  
+  function sendBulkLogs(){
+    if(arrMsg.length == 0){
+      return;
+    }
+    requestOptions.body = arrMsg.toString().replace(/,{/g , "\n{");        // Add New line on the message string
```

Comment:
This should really be implemented as a `\n` delimited JSON stream which is then piped to the outbound request. See [stream-array](https://www.npmjs.com/package/stream-array).

For example: 

```

var streamify = require('stream-array');
var jsonstream = require('jsonstream');
var stringify = jsonstream.stringify(false);

stringify.pipe(process.stdout);

streamify([
  { ok: true },
  { ok: true },
  { ok: true },
  { ok: true },
  { ok: true }
]).pipe(stringify);
```

Outputs:

```
{"ok":true}
{"ok":true}
{"ok":true}
{"ok":true}
{"ok":true}
```